### PR TITLE
Rework docs content for EAP2

### PR DESCRIPTION
**Pull request by eskatos**

### CODE REVIEW

File: docs/EAP2.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Declarative Gradle - November 2024 Update
+
+!!! info
```
Comment (oleg-nenashev) [inline docs/EAP2.md:3]: We can embed the post later if needed.
I have embeddable version for the newsletter, and that code can be ported

File: docs/ROADMAP.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ All the dates and the scope are aspirational.
 To discuss the roadmap and the related initiatives, use the
 `#declarative-gradle` channel on the [Gradle Community Slack](https://gradle.org/slack-invite)
 
-## First Early Access Preview (EAP)
+## First Early Access Preview (EAP) - July 2024
```
Comment (oleg-nenashev) [inline docs/ROADMAP.md:10]: NIT in this context, but it is better to not change the headers as it is used to generate anchor URL. 
I would recommend adding a list of milestones in the top of the page, and keep the target dates there

File: docs/ROADMAP.md [hunk 3]
```diff
@@ -18,10 +18,13 @@ To discuss the roadmap and the related initiatives, use the
...
-## Subsequent Early Access Previews
 
-* 2024-H2
-* More EAPs towards the end of the year
-* More features
-* Addressing collected feedback
-* Further feedback from the community
+## Second Early Access Preview (EAP) - November 2024
+
+## Third Early Access Previes (EAP) - February 2025
```
Comment (oleg-nenashev) [inline docs/ROADMAP.md:24]:
```suggestion
## Third Early Access Preview (EAP) - February 2025
```

File: docs/ROADMAP.md [hunk 4]
```diff
@@ -18,10 +18,13 @@ To discuss the roadmap and the related initiatives, use the
...
+
+## Incubating Software Types and Declarative Configuration Language (DCL) - 2025
```
Comment (oleg-nenashev) [inline docs/ROADMAP.md:26]: Would it make sense to use a common term, e.g. "Incubating/Beta" and "GA" release, and stack items there?

File: docs/publications/2024-09-droidcon-nyc.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (oleg-nenashev) [inline docs/publications/2024-09-droidcon-nyc.md:1]: I just added a link in #138 , but this approach is better

File: docs/ROADMAP.md (hunk 4 above)
Comment (eskatos) [reply docs/ROADMAP.md:26]: I think we should use the Gradle's terminology.
We can link to https://docs.gradle.org/current/userguide/feature_lifecycle.html to make things clearer

File: docs/EAP2.md (hunk 1 above)
Comment (eskatos) [reply docs/EAP2.md:3]: Just like EAP.md, this is a redirect, see mkdocs.yml

File: docs/ROADMAP.md (hunk 3 above)
Comment (eskatos) [reply docs/ROADMAP.md:24]: Yep, I have a fix for this typo locally already

**Review by tresat — APPROVED** [review]

File: docs/getting-started/samples.md [hunk 6]
```diff
@@ -30,6 +30,15 @@ Please follow the README available at [gradle/declarative-samples-kotlin-app](ht
 
 ## Android Application
 
+### Using the Official Android Software Types (Preview)
+
+A sample Android application written in the Declarative Gradle DSL, using the official Android Software Types Preview `androidApplication` and `androidLibrary` defined in the `com.android.ecosystem` ecosystem plugin.
```
Comment (h0tk3y) [inline docs/getting-started/samples.md:35]: `androidApplication` → `androidApp`, unless something has changed.

**Review by big-guy — APPROVED** [review]

File: docs/getting-started/samples.md (hunk 6 above)
Comment (eskatos) [reply docs/getting-started/samples.md]: Thanks! fixed

**Pull request merged**